oam/install_scripts/mariadb-columnstore-start.sh

```diff
@@ -9,6 +9,7 @@
 /bin/systemctl start mcs-exemgr
 /bin/systemctl start mcs-dmlproc
 /bin/systemctl start mcs-ddlproc
+sleep 2
```

Comment:
Adding this until refactor of systemd units to prevent potential races between mcs and dbbuilder